src/main/resources/templates/smallfull/review.html

```diff
@@ -5,1448 +5,1582 @@
       layout:decorate="~{layouts/baseLayout}">
 
 <head>
-  <title>Check your answers</title>
+    <title>Check your answers</title>
```

Comment:
This page seems to have been effected by formatting quite a bit, might be worth looking at!

Reply:
I toroughly went through the page and i believe it is all fine as the IDE just formatted the code